### PR TITLE
Example use for documentation

### PR DESCRIPTION
**Pull request by whyboris**
I'm loving how useful and slim this package is ❤️ 

I thought minor documentation would be helpful 🙇

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,28 @@ It provides a function that takes two string arguments and returns a hash like t
 ````
 
 Please see [tests](./test/test.js) for more insights.
```
Comment (tad-lispy) [inline README.md]: I think it would make sense to move it below the example you provided.

File: README.md [hunk 2]
```diff
@@ -15,3 +15,28 @@ It provides a function that takes two string arguments and returns a hash like t
...
+
+
+### Example
+
+Here is how you can use this package in an _Angular_ project:
```
Comment (tad-lispy) [inline README.md]: It seems to me that there is nothing specific to Angular in this example. Am I missing something?

Comment (tad-lispy) [reply README.md]: How about:

## Install

```sh
npm install damerau-levenshtein
```

## Use with ES6 modules

```js
import * as levenshtien from 'damerau-levenshtein';

// ...

const lev = levenshtien('hello world', 'Hello World!');
// { steps: 4, relative: 0.3076923076923077, similarity: 0.6923076923076923 }
```

## Use with TypeScript

```ts
interface LevenshteinResponse {
  steps: number;
  relative: number;
  similarity: number;
}
```

File: README.md [hunk 3]
```diff
@@ -15,3 +15,28 @@ It provides a function that takes two string arguments and returns a hash like t
...
+
+2. Import into your project
+    `import * as levenshtien from 'damerau-levenshtein';`
+
+3. Use wherever
+    `const lev = levenshtien('hello world', 'Hello World!');`
+
+*Optional*
+
+If you're using _TypeScript_, create an interface
```
Comment (tad-lispy) [inline README.md]: Would you please provide more complete example? How would I use this interface?

**Review by tad-lispy — COMMENTED** [review]
Hey! Thank you very much. It's a change in a right direction. I'd like to suggest few improvements to your improvements... if I may say so 😉 

Also, just like in #12, please update the changelog and take credit for your work.

It's alright if you have no time for this. If that's the case just let me know and I'll take care of it.

**Comment by whyboris** [comment]
Thank you for detailed feedback. What do you think of this? 😄

**Review by tad-lispy — APPROVED** [review]
Good to go. Thanks!

**Comment by tad-lispy** [comment]
Hey, since you are using it with TypeScript, would you mind adding type annotations. If you do please make a separate PR and use the `index.d.ts` file. I'm mot using TypeScript so I'd prefer to stay with vanilla JS for the main module.

Of course only if you have time. You helped a lot already.

**Comment by whyboris** [comment]
This will be a great learning experience -- I've been meaning to learn how all the `.d.ts` files work 😅 

I'll look into it 👍

**Comment by tad-lispy** [comment]
I've published your stuff: https://www.npmjs.com/package/damerau-levenshtein/v/1.0.5

Thank you very much. I'm looking forward for the TS PR.

**Comment by whyboris** [comment]
Thank you for the suggestion, hope this works: #14

**Pull request merged**